config.py

```diff
@@ -290,6 +290,40 @@ def validate_config(CONFIG: CONFIG_DICT_TYPE) -> None:
                   f"{filter_type} is not a valid value for {filter_option} (formerly delay_after_decline) parameter. "
                   f"Choices are: {', '.join(FilterType)}.")
 
+    selection_choices = {"polyglot": ["weighted_random", "uniform_random", "best_move"],
+                         "chessdb_book": ["all", "good", "best"],
+                         "lichess_cloud_analysis": ["good", "best"],
+                         "online_egtb": ["good", "best", "suggest"]}
+    for db_name, valid_selections in selection_choices.items():
+        is_online = db_name != "polyglot"
+        db_section = (CONFIG["engine"].get("online_moves") or {}) if is_online else CONFIG["engine"]
+        db_config = db_section.get(db_name)
+        select_key = "selection" if db_name == "polyglot" else "move_quality"
+        selection = db_config.get(select_key)
+        select = f"{'online_moves:' if is_online else ''}{db_name}:{select_key}"
+        config_assert(selection in valid_selections,
+                      f"'{selection}' is not a valid 'engine:{select}' value. "
```

Comment:
The single quotes in `'{selection}'` and `'engine:{select}'` should probably change to backticks like on line 313 for consistency. Or change the backticks in line 313 to singe quotes.